.gitignore

```diff
@@ -162,3 +162,9 @@ coverage.xml
 
 # Local CMake build config
 cmake_options.yml
+contrib/clojure-package/examples/bert/.lein-failures
```

Comment:
are these necessary here or can they be added to the `.gitignore` under `contrib/clojure-package/examples/bert/`?

Reply:
thanks I'll do that :)